src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
My last response here: this is why (PHP) world sucks. Well, from my point of view "int" is the mistake, even if there are more occurence for that (although I can't see any proof of that). I showed you several arguments why I think using sane type names make sense, you're only arguments are that "everyone uses it" and "int is the type" (which is WRONG).

@brandonsavage please lock this conversation if possible. This became a flame war rather than a sane discussion with arguments.